functorch/csrc/dim/dim.cpp

```diff
@@ -1439,12 +1440,15 @@ py::object getname(PyCodeObject* code, _Py_CODEUNIT c) {
         case STORE_GLOBAL:
           names = code->co_names;
           break;
+#if PY_VERSION_HEX < 0x030b0000
+        // TODO: Re-enable this for Python-3.11
```

Comment:
`PyCode_GetVarnames` and `PyCode_GetCellVars` were added. By changing names to a `py::object` to retain ownership, this fix is a couple lines.

Reply:
Thank you for the feedback. I think a bit more fundamental problem, is that code[GetLasti(frame]) returns some strange opcodes (although dis shows that it still the same)

Reply:
Actually, this might be because we are testing everything using beta1 build of 3.11, let's see if updating to rc2 will make a difference: https://github.com/pytorch/builder/pull/1149